lib/home_page.dart

```diff
@@ -23,27 +36,55 @@ class HomePage extends StatelessWidget {
                   radius: Constants.radiusAvatar,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.name,
                   style: TextStyles.nameTextStyle,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.job,
                   style: TextStyles.jobTextStyle,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.mail,
                   style: TextStyles.mailTextStyle,
                 ),
               ),
+              Padding(
+                padding: const EdgeInsets.all(Constants.padding),
+                child: Row(
```

Comment:
You could change the right and left padding from this section and use the attribute of the Row named mainAxisAlignment: MainAxisAlignment to set a more dynamic position.

Reply:
done